__tests__/app.test.js

```diff
@@ -34,3 +34,33 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id", () => {
+  test("returns coreect array of an review object that matches review ID ", () => {
+    const expected = {
+      review_id: 2,
+      title: "Jenga",
+      designer: "Leslie Scott",
+      owner: "philippaclaire9",
+      review_img_url:
+        "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+      review_body: "Fiddly fun for all the family",
+      category: "dexterity",
+      created_at: "2021-01-18T10:01:41.251Z",
+      votes: 5,
+    };
+    return request(app)
+      .get("/api/reviews/2")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        expect(Object.keys(body)).toEqual(["review"]);
+
+        const { review } = body;
+        expect(review).toBeInstanceOf(Array);
+        expect(review).toHaveLength(1);
+
+        expect(review[0]).toEqual(expected);
+      });
```

Comment:
Instead of toEqual please use a more 'flexible' test like 'objectContaining' or 'toMatchObject' to give this endpoint space for extension in the future. toEqual is an example of brittle testing